mkt/reviewers/urls.py

```diff
@@ -71,6 +72,8 @@
     url(r'reviewers/', include(reviewers_router.urls)),
     url('^reviewers/search', views.ReviewersSearchView.as_view(),
         name='reviewers-search-api'),
+    url('^reviewers/sites/search', WebsiteSearchView.as_view(),
```

Comment:
oh, yeah, had not caught the import, that was surprising. Maybe rename to `ReviewersWebsiteSearchView` yeah, including in `views.py` so that you don't need to import as a different name ?